src/sentry/features/__init__.py

```diff
@@ -123,7 +123,8 @@
 default_manager.add("organizations:performance-ops-breakdown", OrganizationFeature)  # NOQA
 default_manager.add("organizations:performance-tag-explorer", OrganizationFeature)  # NOQA
 default_manager.add("organizations:team-alerts-ownership", OrganizationFeature)  # NOQA
-
+default_manager.add("organizations:external-user-associations", OrganizationFeature)  # NOQA
+default_manager.add("organizations:external-team-associations", OrganizationFeature)  # NOQA
```

Comment:
Did you read the comment directly below this lol

Reply:
argh, I missed the sort alphabetically!